### PR TITLE
light emiting material

### PR DESCRIPTION
**Pull request by bartektempka**

### CODE REVIEW

**Review by wojciechloboda — APPROVED** [review]
koks

File: src/main.cu [hunk 1]
```diff
@@ -128,7 +126,7 @@ int main()
     //create_world
 
     // Load object
-    const char *file_path = "models/cube.obj";
+    const char *file_path = "models/cubes.obj";
```
Comment (pklatka) [inline src/main.cu:129]: Let's revert that change, not relevant in this PR and might be problematic for my local setup 😬

File: src/camera.h [hunk 2]
```diff
@@ -71,20 +73,25 @@ class camera
...
                 //background
-                float3 unit_direction = normalize(cur_ray.direction());
-                float t = 0.5f*(unit_direction.y + 1.0f);
-                float3 c = (1.0f-t)*make_float3(1.0, 1.0, 1.0) + t*make_float3(0.5, 0.7, 1.0);
-                return cur_attenuation * c;
+                return cur_attenuation*background_color;
+                // float3 unit_direction = normalize(cur_ray.direction());
+                // float t = 0.5f*(unit_direction.y + 1.0f);
+                // float3 c = (1.0f-t)*make_float3(1.0, 1.0, 1.0) + t*make_float3(0.5, 0.7, 1.0);
+                // return cur_attenuation * c;
```
Comment (pklatka) [inline src/camera.h:94]: Same here :)

File: src/camera.h [hunk 3]
```diff
@@ -71,20 +73,25 @@ class camera
...
-                    cur_attenuation *= attenuation;
+                    cur_attenuation *= attenuation + color_from_emission;
                     cur_ray = scattered;
                 }
                 else {
-                    return make_float3(0.0,0.0,0.0);
+                    cur_attenuation *= color_from_emission;
+                    return cur_attenuation;
+                    
+                    // return make_float3(0.0,0.0,0.0);
```
Comment (pklatka) [inline src/camera.h:85]: Hmm I guess this can be removed, but let me know if I'm wrong :)

**Review by pklatka — CHANGES_REQUESTED** [review]

**Comment by pklatka** [comment]
Also, any idea why I see a dark screen when running the program? 

image

File: src/main.cu (hunk 1 above)
Comment (bartektempka) [reply src/main.cu:129]: yikes

File: src/camera.h (hunk 2 above)
Comment (bartektempka) [reply src/camera.h:94]: Kinda want to leave it in cus lights to be bright need a lot of samples per pixel this background gives us soft iluminatio much faster.

File: src/camera.h (hunk 3 above)
Comment (bartektempka) [reply src/camera.h:85]: no it cannot

Comment (pklatka) [reply src/camera.h:85]: I'm talking about this line:
```
 // return make_float3(0.0,0.0,0.0);
```
If there is no information like "Comment on lines +91 to +94" this means that this comment refers to the first bottom line which is visible, I guess you could've miss that. Anyway, we don't need to remove that, we can get rid of it at a later point.

File: src/main.cu (hunk 1 above)
Comment (bartektempka) [reply src/main.cu:129]: hmmm with 1 cube is hard to see the effect of light

**Pull request merged**